streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -532,6 +582,48 @@ public void verifyStore() {
         );
     }
 
+    private <T> void shouldHandleWindowKeyQueries(final Integer key, final Instant timeFrom, final Instant timeTo, final Function<T, Integer> extractor) {
+        shouldHandleWindowKeyQuery(
+                key,
+                timeFrom,
+                timeTo,
+                extractor,
+                mkSet(1, 2, 3)
+
+        );
+    }
+
+    private <T> void shouldHandleWindowRangeQueries(final Function<T, Integer> extractor) {
+        shouldHandleRangeQuery(
```

Comment:
Are these calling the right method?

```suggestion
        shouldHandleWindowRangeQuery(
```